tareas/clase-5/tarea-4/tarea-4-clase-5.js

```diff
@@ -0,0 +1,17 @@
+document.querySelector('#calcular-promedio').onclick = function () {
+    const listaDeNumeros = document.querySelectorAll('.numero');
+    let acumulador = 0;
+    for (let i = 0; i < listaDeNumeros.length; i++) {
+        const numero = Number(listaDeNumeros[i].innerText);
+        console.log(numero);
+        acumulador += numero;
+
+        document.querySelector(
+            '.suma-numeros-total',
+        ).innerText = `La suma total de todos los numeros es: ${acumulador}`;
+
+        const promedio = acumulador / listaDeNumeros.length;
+
+        document.querySelector('.promedio').innerText = `El promedio es: ${promedio}`;
```

Comment:
estás repitiendo esto dentro del for, en cada iteración vas a cambiar el texto de suma-numeros-total, en cada iteración vas a calcular el promedio, en cada iteración vas a cambiar el texto de .promedio.

Todo eso va FUERA del for, se hace una única vez, al final.